src/eterna/AnnotationManager.ts

```diff
@@ -1042,15 +1020,15 @@ export default class AnnotationManager {
             const view = this.getAnnotationView(params.pose, i, params.item);
             if (params.item.type === AnnotationHierarchyType.ANNOTATION) {
                 view.onMovedAnnotation.connect((point: Point) => {
-                    const anchorIndex = this._annotations[params.itemIndex].positions[i].anchorIndex;
+                    const anchorIndex = this.allAnnotations[params.itemIndex].positions[i].anchorIndex;
                     const base = params.pose.getBase(anchorIndex);
                     const anchorPoint = new Point(
                         base.x + params.pose.xOffset,
                         base.y + params.pose.yOffset
                     );
                     // Compute relative position
                     const movedPosition: AnnotationPosition = {
-                        ...this._annotations[params.itemIndex].positions[i],
+                        ...this.allAnnotations[params.itemIndex].positions[i],
```

Comment:
`item` vs `this.allAnnotations[params.itemIndex]`

Reply:
Good catch